v2/robotmk/scheduler.py

```diff
@@ -148,7 +154,10 @@ def __init__(self, suite_specification: _SuiteSpecification) -> None:
 
     def __call__(self) -> None:
         retry_spec = RetrySpec(
-            id_=uuid4(),
+            identifier=Identifier(
+                name=self._suite_spec.name,
+                timestamp=datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S"),
```

Comment:
Could we get clashes here, if somebody updates there local time? Should we maybe fallback to utc time here?

OTI: I think just using ISO8601 would be fine here: https://stackoverflow.com/questions/2150739/iso-time-iso-8601-in-python

Reply:
I actually first tried the ISO format, but ":" is not allowed in Windows directory names. I now use the ISO format with ":" replaced by ".".

Reply:
Ah, now that I understand the reasoning, I prefer the previous version :smile: But the important part was the timezone anyway. Let's not iterate further on this version, they don't seem important at the moment.